x-pack/plugins/case/server/routes/api/cases/comments/delete_all_comments.ts

```diff
@@ -20,6 +21,11 @@ export function initDeleteAllCommentsApi({ caseService, router, userActionServic
         params: schema.object({
           case_id: schema.string(),
         }),
+        query: schema.maybe(
+          schema.object({
+            subCaseID: schema.maybe(schema.string()),
```

Comment:
I think the `case_id` is sufficient. We can get the case and check its type. If it a parent case we are done. If it is a sub case we get the id of the parent case and we proceed accordingly. What do you think?

Reply:
I talked to Xavier about this, we're worried about if we just use `/case/<id>/comments` we might run into issues if we get an SDH because it won't be clear if the request was meant for a Case or sub case. A potential solution down the road would be something like `/case/sub_case/<id>/comments` that way it's clear. Let's talk about this tomorrow though 👍